SKILL.tsv

```diff
@@ -4067,8 +4067,8 @@ SKILL_20150428_004066	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds{nl}U
 SKILL_20150428_004067	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds{nl}Use 1 Gather Corpse
 SKILL_20150428_004068	$Hanging Shot
 SKILL_20150428_004069	$Guardian Saint duration: #{CaptionTime}# seconds{nl}Chance on hit: 50%{nl}Charges: #{CaptionRatio}#
-SKILL_20150428_004070	$Creates a magic circle in front of you that increases the DEX of allies and decreases the enemy's evasion and physical defense. Party members will receive the effect without stepping on the magic circle.
-SKILL_20150428_004071	$Gives a blessing providing additional damage to the attacks of your party members and allies.
+SKILL_20150428_004070	Create a magic circle in front of you that increases the dexterity of allies whilst decreasing the enemy's evasion and physical defense. Party members will receive the effect without stepping on the magic circle.
```

Comment:
No need to change DEX to dexterity. 

Create a magic circle in front of you that increases DEX of allies whilst decreasing the enemy's evasion and physical defense. Party members will receive the effect without stepping on the magic circle.